inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,65 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Restaurant {
+  private String name;
+  private double stars;
+  private String priceCategory;
+  private ArrayList<Review> reviews = new ArrayList<>();
+
+  Restaurant(String name, double stars, String priceCategory) {
+    this.name = name;
+    if (stars > 5) {
+      this.stars = 5;
+    } else {
+      this.stars = stars;
+    }
+    this.priceCategory = priceCategory;
+  }
+
+  public String getName() {
+    return this.name;
+  }
+
+  public double getStars() {
+    return this.stars;
+  }
+
+  public String getPriceCategory() {
+    return this.priceCategory;
+  }
+
+  public void addReview(Review review) {
+    this.reviews.add(review);
+    updateStars();
+  }
+
+  public ArrayList<Review> getReview() {
```

Comment:
Since the variable is `reviews` this should be called `getReviews` (with an 's')